lib/services/kinvey-user-service.ts

```diff
@@ -0,0 +1,44 @@
+import { join } from "path";
+import { home } from "osenv";
+
+import { UserServiceBase } from "./user-service-base";
+
+export class KinveyUserService extends UserServiceBase implements IUserService {
+	constructor($injector: IInjector,
+		private $hostInfo: IHostInfo) {
+		super($injector);
+		this.userFilePath = this.getUserFilePath();
+	}
+
+	public getUserData(): IUserData {
+		const userData: IKinveyUserData = <any>super.getUserData();
```

Comment:
You can set the `getUserData` method's return type to `any` in order to avoid the cast